olive/model/__init__.py

```diff
@@ -670,7 +670,13 @@ def get_io_config(self) -> Dict[str, Any]:
         elif self.hf_config and self.hf_config.task and not self.hf_config.components:
             # hf_config is provided
             logger.debug("Using hf onnx_config to get io_config")
-            io_config = self.hf_config.get_io_config(self.model_path)
+            # For MLFlow model, get io config from model_name instead of model_path
```

Comment:
Could you help add this into our docs? To enforce user fill the model_name if the model is mlflow format.

Reply:
We need to update a lot for [hf model section](https://microsoft.github.io/Olive/features/huggingface_model_optimization.html#). I will send a different PR for this.